tools/ci_build/github/linux/docker/scripts/install_deps.sh

```diff
@@ -39,7 +39,7 @@ else
   #bae6333e149a59a3faa9c4d9c44974373dcf5256 is v1.3.0
   #9e55ace55aad1ada27516038dfbdc66a8a0763db is v1.4.1
   #3a9a87871021a02d6b970c1e26ffa4c0dd720677 is v1.4.1 latest
-  for onnx_version in "5af210ca8a1c73aa6bae8754c9346ec54d0a756e" "bae6333e149a59a3faa9c4d9c44974373dcf5256" "9e55ace55aad1ada27516038dfbdc66a8a0763db" "3a9a87871021a02d6b970c1e26ffa4c0dd720677"; do
+  for onnx_version in "5af210ca8a1c73aa6bae8754c9346ec54d0a756e" "bae6333e149a59a3faa9c4d9c44974373dcf5256" "9e55ace55aad1ada27516038dfbdc66a8a0763db" "3a9a87871021a02d6b970c1e26ffa4c0dd720677" "fdb978a580bcb461ea077b9772c973928b8dbf5f"; do
```

Comment:
Just replace 3a9a87871021a02d6b970c1e26ffa4c0dd720677 to the latest commit ID. (and change the comment line (L41))